### ISSUE TITLE
Add new well-known attribute

### ISSUE THREAD

**Issue by KirillovDenis** [nspcc-dev/neofs-sdk-go#215]
It would be nice to have new [well-known attribute](https://github.com/nspcc-dev/neofs-sdk-go/blob/master/object/wellknown_attributes.go) `FilePath`. It's used in http-gw, cdn-tool and other apps.

**Comment by alexvanin** [comment]
It seems that these consts are taken from https://github.com/nspcc-dev/neofs-api/blob/431335054c3bd2fcff52226e04175b443bcb8a1e/object/types.proto#L134-L141, so API change is required.

**Comment by alexvanin** [comment]
> In release we'll provide a method pair for each well-known attribute:
> 
> ```
> func (*Object) SetFilePath(string)
> func (Object) FilePath() string
> // same for other well-known attributes
> ```
> 
> This will allow to write/read this data to/from the object and don't care about attribute details.

Those are well-known attributes, but they are not _system_ attributes (used by core NeoFS) so I would rather avoid such functions at least for v1.0 release. 

We also intend to use this constant in _search_ operation, so attribute name is going to be used as a key in search filter.

And if such method pairs will be implemented, they should  return bool flag if the attribute was set or not, I guess.

**Comment by cthulhu-rider** [comment]
> I would rather avoid such functions at least for v1.0 release.

There are two possible approaches:

1. do not work with these non-system object values in SDK and and leave it on the app-side
2. provide special functions to work with these data in SDK

IMO both approaches are appropriate.

About 2nd: I don't know why applications should care about how filepaths are written in object structurally. For search filters there will be a function like `func SelectByFilepath(fs *SearchFilters, path string)`

**Comment by carpawell** [comment]
Unblocked after https://github.com/nspcc-dev/neofs-api/pull/239.
